src/pages/AddPage/AddPage.jsx

```diff
@@ -16,9 +18,18 @@ const AddPage = () => {
         body: JSON.stringify({ text }),
       });
       setIsSuccess(true);
+      setError("");
+      setIsSuccess("");
+      setText("");
+      setTodosCount((prevCount) => ({
+        ...prevCount,
+        all: prevCount.all + 1,
+        uncompleted: prevCount.uncompleted + 1,
```

Comment:
either `completed` or `uncompleted`. The initial state is something else. Find all the places that you are changing/defining the `todosCount` state.